boot.js

```diff
@@ -26,10 +26,26 @@ function wrap (server, opts, instance) {
   if (server[readyKey]) {
     throw new Error(readyKey + '() is already defined, specify an expose option')
   }
-
   server[useKey] = function (a, b, c) {
     instance.use(a, b, c)
-    return this
+    return {
+      __proto__: this,
```

Comment:
Doesn't this need to be `'__proto__'` (https://tc39.es/ecma262/#sec-__proto__-property-names-in-object-initializers)?

And isn't setting the prototype in this way considered deprecated?

Reply:
Using `Object.create()` will achieve the same thing.

Reply:
I think that would be better.

Reply:
As an object key it's already a string, it doesn't have to be explicitly quoted.

The case for `{__proto__}`: `__proto__` in an object literal is an uncontested legitimate part of the spec (as opposed to the `obj.__proto__` getter/setter property which is contested). Personally I think it's more elegant - although I wouldn't use it in application code I would use it in something like this. `Object.create` will create more boilerplate noise (and will need more objects for the descriptor) but if there's consensus on changing it I'll change it np. @mcollina @delvedor if you weigh in on preference in addition to @jsumners 
 I'll act accordingly

Reply:
> As an object key it's already a string, it doesn't have to be explicitly quoted.

Yeah, but this particular "string" makes me uneasy. I think it is ambiguous as to what the runtime will actually do in this case.

Reply:
I think Object.create() is more idiomatic and readable.

Reply:
I agree with @mcollina.

Reply:
😆 then I defer to the consensus